src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -418,6 +418,12 @@ def exit_host_maintenance(self, hostname: str) -> OrchResult:
         """
         raise NotImplementedError()
 
+    def rescan_host(self, hostname: str) -> OrchResult:
+        """
+        Perform a HBA rescan on a host
+        """
```

Comment:
```suggestion
        """
        Issue a disk rescan on every each HBA.
        
        Some HBA's and external enclosures don't automatically register
        device insert with the kernel, so for these scenarios we need
        to manually recan the hosts HBAs
        
        :param hostname: (str) host name
        """
```

Reply:
(nit) use the same for docstring each impl of this func?

Reply:
wrt mgfritch's suggestion:

s/HBA's/HBAs/
s/insert/insertions/

Reply:
done